sample/src/main/res/layout/activity_survey_list.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingBottom="@dimen/activity_vertical_margin"
```

Comment:
I don't think we need to add the padding here 🤔  The recycler view will not be stretched full screen and the user might not be able to scroll on edge of screens.

Reply:
@hoangnguyen92dn Thanks for your information. Actually, we must follow up the real design, and in this sample, I just try to make all screen consistency.